platforms/esp/32/clockless_esp32.h.orig

```diff
@@ -227,12 +227,12 @@ public:
 
 <<<<<<< HEAD
         gControllers[gNumControllers] = this;
-        gNumControllers++;
+        ++gNumControllers;
 
         mPin = gpio_num_t(DATA_PIN);
 =======
 	gControllers[gNumControllers] = this;
-        gNumControllers++;
+        ++gNumControllers;
```

Comment:
There's currently a merge conflict here, which would need to be addressed.

Reply:
That one is not on me ;) that file exists for two years now. I could remove the changes from that file though. That file itself should probably get removed completely though.